features/article.yml

```
@@ -0,0 +1,6 @@
+name: <article>
+description: The `<article>` element is a self-contained composition in a document, page, application, or site, which is intended to be independently distributable or reusable.
```

Making a suggestion here to trade in some concision to make room for illustrating something abstract with concrete examples:

```suggestion
description: The `<article>` element represents self-contained content, which is intended to be independently distributable or reusable, such as a comment, a blog post, or news article.
```